### PR TITLE
Autoprefixer + Flex

### PR DESCRIPTION
**Pull request by gvn**
R? @gesa

### CODE REVIEW

**Comment by gesa** [comment]
YESSSSSSS. Autoprefixer is the best! (code looks great, pulling in to take a look in browser.)

File: dist/css/webmakerLogin.css [hunk 1]
```diff
@@ -354,8 +360,17 @@
     width: 70%;
   }
 }
+.modal-content .modal-body .checkbox label {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
```
Comment (gesa) [inline dist/css/webmakerLogin.css]: honestly if we're using Autoprefixer now, this can all go, yeah?

**Comment by gesa** [comment]
R+! :clap:

**Comment by gvn** [comment]
:tada: :balloon:

**Pull request merged**